package/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java

```diff
@@ -186,9 +216,23 @@ private void updateAll() {
       mStepCalculated = (mMaxValue - mMinValue) / (double) DEFAULT_TOTAL_STEPS;
     }
     setMax(getTotalSteps());
+    updateLowerLimit();
+    updateUpperLimit();
     updateValue();
   }
 
+  /** Update limit based on props limit, max and min */
+  private void updateLowerLimit() {
+    double limit = Math.max(mRealLowerLimit, mMinValue);
+    mLowerLimit = (int) Math.round((limit - mMinValue) / (mMaxValue - mMinValue) * getTotalSteps());
```

Comment:
The reason behind all these calculations is because the method `onProgressChanged` that the SeekBar change listener triggers gives us the SeekBar progress in the range of 0 to 20. So we translate all the Slider values to SeekBar values and be able to use them here https://github.com/callstack/react-native-slider/pull/432/files#diff-dc644f156dbc97ab8c8e17dcd152efba4e8b761399400b27eddba3a87640521bR45.

Reply:
Thanks for explanation! 👍